test/Npgsql.Tests/ConnectionTests.cs

```diff
@@ -1446,8 +1443,8 @@ public void PasswordSourcePrecendence()
             {
                 builder.Password = password;
                 builder.Passfile = passFile;
-                builder.Pooling = false;
                 builder.IntegratedSecurity = false;
+                builder.ApplicationName = $"{nameof(PasswordSourcePrecendence)}:{Guid.NewGuid()}";
```

Comment:
This is good, but take a look at CreateTempPool which also clears the pool after the test runs. The idea is to avoid having single-use pools (with special connection strings) where an idle connector exists forever in the tests.